app/src/app/components/contents/PacketGroup/packetColumns.tsx

```diff
@@ -1,13 +1,43 @@
-import { createColumnHelper } from "@tanstack/react-table";
+import { createColumnHelper, SortDirection } from "@tanstack/react-table";
+import { ArrowDown, ArrowUp, ArrowUpDown } from "lucide-react";
 import { Link } from "react-router-dom";
 import { Packet } from "../../../../types";
+import { Button } from "../../Base/Button";
+import { FilterInput } from "../common/FilterInput";
 import { ParameterContainer } from "../common/ParameterContainer";
 
-const columnHelper = createColumnHelper<Packet>();
+const SortIcon = ({ sortedDirection }: { sortedDirection: false | SortDirection }) => {
+  if (!sortedDirection) return <ArrowUpDown className="ml-2 h-4 w-4" />;
+  return sortedDirection === "asc" ? <ArrowUp className="ml-2 h-4 w-4" /> : <ArrowDown className="ml-2 h-4 w-4" />;
+};
 
+const columnHelper = createColumnHelper<Packet>();
 export const packetColumns = [
   columnHelper.accessor("id", {
-    header: "Packet",
+    header: ({ column }) => {
+      const sortedDirection = column.getIsSorted();
+
+      return (
+        <div className="flex flex-col items-start space-y-1 mb-2">
+          <Button
+            variant="ghost"
+            size="sm"
+            className="sm:w-36 lg:w-48 justify-start"
```

Comment:
To my eye, this button and the 'Parameters' div take up too much vertical space - particularly the 'Parameters' text which is further from the input below it than the button is.

I suggest adding `h-fit py-1.5` to each, and make sure both div containers have the same `space-y-[n]` class (they differ).

Because of the background color appearing when the button is hovered, the containers will also need a little padding on top.